src/pages/main/components/selected-dataset/build-flow/components/cards/description-card/description-card.tsx

```diff
@@ -0,0 +1,169 @@
+import styles from './description-card.module.css'
+
+import { useEffect, useState } from 'react'
+import { useHistory } from 'react-router'
+import cn from 'classnames'
+
+import { ExploreGenomeTypes } from '@core/enum/explore-genome-types-enum'
+import { getApiUrl } from '@core/get-api-url'
+import { t } from '@i18n'
+import { datasetStore } from '@store/dataset'
+import dirinfoStore from '@store/dirinfo'
+import filterStore from '@store/filter'
+import { Button } from '@ui/button'
+import { Card } from '@ui/card'
+import { Icon } from '@ui/icon'
+import { GlbPagesNames } from '@glb/glb-names'
+import { showToast } from '@utils/notifications/showToast'
+import {
+  exploreCandidateOptionsList,
+  optionsForOpenButton,
+} from '../../wizard/wizard.data'
+import { ICardProps } from '../../wizard/wizard.interface'
+import wizardStore from '../../wizard/wizard.store'
+import { getNextPageData, memorizeLocation } from '../../wizard/wizard.utils'
+import { CardTitleWithEdit } from '../components/card-edit-title'
+import { CardRadioList } from '../components/card-radio-list'
+
+const descriptions: { [key: string]: string } = {}
+let typingTimer: ReturnType<typeof setTimeout>
```

Comment:
No complex objects should exist in tsx files. The only thing is allowed to store -- simple constants with primitives that are only used inside this file.
Create a local description-card.store.ts and move all this stuff there.